spa/src/components/account/SignIn/SignInForm.js

```diff
@@ -0,0 +1,85 @@
+import { useState } from 'react';
+import * as S from '../Account.styled';
+import { useForm } from 'react-hook-form';
+
+import visibilityOn from 'assets/images/account/visibility_on.png';
+import visibilityOff from 'assets/images/account/visibility_off.png';
+
+import { FORGOT_PASSWORD } from 'routes';
+
+function SignInForm({ onSubmitSignIn, loading }) {
+  const [showPassword, setShowPassword] = useState(false);
+  const togglePasswordVisiblity = () => {
+    setShowPassword(showPassword ? false : true);
```

Comment:
If you don't want to use `useModal`, better to use previous state to toggle. This current implementation could add some unwanted bugs.
```suggestion
    setShowPassword(prevShow => !prevShow);
```